extensions/amp-date-countdown/amp-date-countdown.md

```diff
@@ -0,0 +1,134 @@
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>fill, fixed, fixed-height, flex-item, nodisplay, responsive</td>
+  </tr>
+</table>
+
+## Behavior
+
+The `amp-date-countdown` provides countdown time data that you can render in your AMP page. By providing specific [attributes](#attributes) in the  `amp-date-countdown` tag, the `amp-date-countdown` extension returns a list of time parameters, which you can pass to an `amp-mustache` template for rendering.  Refer to the [ list below for each returned time parameter](#returned-time-parameters).
+
+![count-down](https://user-images.githubusercontent.com/2099009/28486908-71f03336-6e3c-11e7-9822-3bac6528b148.png)
+
+### Returned time parameters
+
+This table lists the format you can specify in your Mustache template:
+
+Format | Meaning
+-- | --
+d | day - 0, 1, 2,...12, 13..Infinity
+dd | day - 00, 01, 02, 03..Infinity
+h | hour - 0, 1, 2,...12, 13..Infinity
+hh | hour - 01, 02, 03..Infinity
+m | minute - 0, 1, 2,...12, 13..Infinity
+mm | minute - 01, 01, 02, 03..Infinity
+s | second - 0, 1, 2,...12, 13..Infinity
+ss | second - 00, 01, 02, 03..Infinity
+days | i18n string for day or days
+hours | i18n string for hour or hours
+minutes | i18n string for minute or minutes
+seconds | i18n string for second or seconds
+
+### Examples
+
+This table provides examples of formatted values specified in a Mustache template, and a sample of what the output:
+
+Format | Sample Output | Remarks
+-- | -- | -
+{hh}:{mm}:{ss} | 04:24:06 | -
+{h} {hours} and {m} {minutes} and {s} {seconds} | 4 hours and 1 minutes and 45 seconds | -
+{d} {days} {h}:{mm} | 1 day 5:03 | -
+{d} {days} {h} {hours} {m} {minutes} | 50 days 5 hours 10 minutes | -
+{d} {days} {h} {hours} {m} {minutes} | 20 days 5 hours 10 minutes | -
+{h} {hours} {m} {minutes} | 240 hours 10 minutes | `biggest-unit='hours'`
+{d} {days} {h} {hours} {m} {minutes} | 50 天 5 小时 10 分钟 | `locale='zh-CN'`
+
+
+## Attributes
+
+You must specify at least one of these required attributes: `end-date`, `timestamp-ms`, `timestamp-seconds`.
+
+##### end-date
+An ISO formatted date to count down to. For example, `2020-06-01T00:00:00+08:00`
+
+##### timestamp-ms
+A POSIX epoch value in milliseconds; assumed to be UTC timezone. For example, `timestamp-ms="1521880470000"`.
+
+##### timestamp-seconds
+A POSIX epoch value in seconds; assumed to be UTC timezone. For example, `timestamp-seconds="1521880470"`.
+
+##### offset-seconds (optional)
+A positive or negative number that represents the number of seconds to add or subtract from the `end-date`. For example, `offset-seconds="60"` adds 60 seconds to the end-date.
+
+##### when-ended (optional)
+Specifies whether to stop or continue the timer when it reaches 0 seconds. The value can be set to `stop` (default) or `continue`. If `stop`, the timer stops at 0 seconds and will not pass the final date.
```

Comment:
Please provide details on what the `continue` value does and what it's good for.

Reply:
actually we do not have any special logic for `continue`, do you think we need to change the entire attribute name from `when-ended` to `when-ended-stop` = `yes|no` instead ?